bedrock/mozorg/templates/mozorg/home/home-quantum.html

```diff
@@ -76,7 +76,7 @@
   <section class="intro-section" id="mozilla">
     <div class="content">
       <div class="intro-image">
-        <img src="{{ static('img/home/quantum/dino.png') }}" alt="" />
+        <img src="{{ static('img/home/quantum/dino.svg') }}" alt="" />
```

Comment:
I think this needs width and height attributes. Chrome and IE are showing the dino tiny for some reason, maybe some default sizing when size can't be calculated from the image data. So the SVG file probably needs width and height attributes as well, since it looks like some browsers don't grok viewBox. Firefox shows it at the viewBox size on wide desktop sizes, which is probably too big (the `max-size: 100%;` kicks in and scales it down at smaller breakpoints).